src/workerd/api/crypto/aes.c++

```diff
@@ -408,7 +408,8 @@ class AesCbcKey final: public AesKeyBase {
     int plainSize = 0;
     auto blockSize = EVP_CIPHER_CTX_block_size(cipherCtx.get());
 
-    auto plainText = kj::heapArray<kj::byte>(cipherText.size() + ((blockSize > 1) ? blockSize : 0));
+    KJ_STACK_ARRAY(
+        kj::byte, plainText, cipherText.size() + ((blockSize > 1) ? blockSize : 0), 1024, 4096);
```

Comment:
Can you move the magical numbers to their unique declarations with appropriate comments?

Reply:
As much as I generally agree with you, I think it's better to keep usage consistent and in every other place we use `KJ_STACK_ARRAY` we inline the sizes directly the same way.

Reply:
At least can you add comments explaining what those magical numbers are?

Reply:
That's explained in the doctoring of KJ_STACK_ARRAY though

Reply:
Sorry if I'm not clear enough: I know what does numbers represent. I don't know why we chose those numbers. That's what I meant by the magical number usage.